packages/dx-react-grid/docs/reference/table-header-row.md

```diff
@@ -64,6 +64,15 @@ Field | Type | Default | Description
 ------|------|---------|------------
 sortingHint? | string | 'Sort' | Specifies the 'Sort' hint's text. Available in the "@devexpress/dx-react-grid-material-ui" package.
 
+## Plugin Components
+
+Name | Properties | Description
+-----|------------|------------
+TableHeaderRow.Row | [TableRowArgs](table.md#table-row-args) | A component that renders a header row.
```

Comment:
The 'TableHeaderRow' namespace seems a bit weird. Can we avoid it?

Reply:
I think it makes docs more readable.

Reply:
For me, it is not quite clear from this sentence that Row and Cell are static members of TableHeaderRow.